components/collectionDetailsPopover/CollectionDetailsPopover.vue

```diff
@@ -13,17 +14,26 @@
       </div>
     </template>
   </tippy>
+
+  <nuxt-link
+    :to="`/${urlPrefix}/collection/${collection?.id}`"
+    class="has-text-link block md:hidden">
+    {{ collection?.name || collection?.id }}
+  </nuxt-link>
```

Comment:
Is it possible to replace this section with `<div class="md:hidden"><slot name="content" /></div>`, as it's more clean?

Reply:
this is for mobile though, the content in mobile is only the collection name, <slot name="content" /> this has the popover content, not the collection name